docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1780 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #80.            if min_dist &lt;= 0: 
</pre><p>Combine this with the <code>if</code> statement directly above via <code>or</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
will revise it.